src/styles/_catalog.scss

```diff
@@ -13,10 +10,23 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
```

Comment:
You can make mixin for media and cretae variable for card's width

Reply:
thanks for review! Mixins is a good idea, did it

> Your catalog block is not centered
in the manual - Blocks should start from the left